test/index.spec.ts

```diff
@@ -26,12 +27,34 @@ describe('IlcSdk', () => {
     describe('constructor options', () => {
         it('should correctly set default options', () => {
             const ilcSdk = new IlcSdk();
-            expect((<any>ilcSdk).log).equal(console);
-            expect((<any>ilcSdk).defaultPublicPath).equal('/');
+            expect((ilcSdk as any).log).equal(console);
+            expect((ilcSdk as any).defaultPublicPath).equal('/');
         });
     });
 
     describe('processRequest', () => {
+        describe('getCurrentReqHost', () => {
```

Comment:
let's add simple test for `getCurrentReqOriginalUri` too

Reply:
added